docker/llm/finetune/lora/cpu/docker/Dockerfile

```diff
@@ -0,0 +1,59 @@
+ARG HTTP_PROXY
+ARG HTTPS_PROXY
+
```

Comment:
use  PIP_NO_CACHE_DIR=false to turn off pip cache and reduce the size of the image
https://github.com/intel-analytics/BigDL/blob/main/docker/llm/inference/cpu/docker/Dockerfile#L5